velox/vector/BaseVector.h

```diff
@@ -157,6 +157,14 @@ class BaseVector {
     return rawNulls_ ? bits::isBitNull(rawNulls_, idx) : false;
   }
 
+  /// Returns true if value at specified index is null or contains null.
```

Comment:
I guess this are the contents of the previous PR? If you're stacking them, you can change the base branch of this PR when you create it to be the branch in the base of the stack instead of main, so it does not show the base changes as part of this PR.

Reply:
> you can change the base branch of this PR when you create it to be the branch in the base of the stack instead of main, so it does not show the base changes as part of this PR.

I'll try that. For now, you can go to Commits tab and select 2nd commit.